### PR TITLE
Handle trivia correctly

### LINKED ISSUE #21948: 'Add Overloads' Quick-Fix for BC40003 produces invalid code
**Version Used**: VS2017 RTM

**Steps to Reproduce**:

1. Drop this in a new console app.

``` VB.NET
Class Base
    Sub M()

    End Sub
End Class

Class Derived
    Inherits Base

    Sub M() ' <--- Invoke the light bulb on this line.

    End Sub
End Class
```

2. Invoke the light bulb on the indicated line.

**Expected Behavior**:
Modifier is either added on same line as `Sub` keyword.

**Actual Behavior**:
Modifier is added to a new line before the `Sub` keyword, which produces parse errors:

image

### PR DESCRIPTION
**Pull request by Youssef1313**
Fixes #21948

I also moved to the new testing framework. (I forgot to make this step a separate commit - but the PR is still easy to review)

### CODE REVIEW

File: src/Features/VisualBasic/Portable/CodeFixes/OverloadBase/OverloadBaseCodeFixProvider.AddKeywordAction.vb [hunk 1]
```diff
@@ -49,10 +50,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeFixes.OverloadBase
 
             Private Async Function GetNewNodeAsync(document As Document, node As SyntaxNode, cancellationToken As CancellationToken) As Task(Of SyntaxNode)
                 Dim newNode As SyntaxNode = Nothing
+                Dim trivia As SyntaxTriviaList = node.GetLeadingTrivia()
+                node = node.WithoutLeadingTrivia()
 
                 Dim propertyStatement = TryCast(node, PropertyStatementSyntax)
                 If propertyStatement IsNot Nothing Then
-                    newNode = propertyStatement.AddModifiers(SyntaxFactory.Token(_modifier))
+                    newNode = propertyStatement.WithoutLeadingTrivia().AddModifiers(SyntaxFactory.Token(_modifier))
```
Comment (Youssef1313) [inline src/Features/VisualBasic/Portable/CodeFixes/OverloadBase/OverloadBaseCodeFixProvider.AddKeywordAction.vb:58]:
```suggestion
                    newNode = propertyStatement.AddModifiers(SyntaxFactory.Token(_modifier))
```

**Comment by Youssef1313** [comment]
Temporarily converting to a draft until the conversation in the linked issue is resolved.

**Comment by Youssef1313** [comment]
@CyrusNajmabadi Can you review please? Thanks.

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
> I also moved to the new testing framework. (I forgot to make this step a separate commit - but the PR is still easy to review)

Not a problem :)  Thanks!

**Pull request merged**